docgen/docs.go

```diff
@@ -30,8 +32,27 @@ func init() {
 }
 
 func main() {
-	// Generate CLI docs
-	if err := doc.GenMarkdownTree(cmd.New(), directory); err != nil {
-		log.Fatalf("Error generating docs: %s", err)
+	mdContent := "# Witness CLI Reference\n\nThis is the reference for the Witness command line tool, generated by [Cobra](https://cobra.dev/).\n\n"
+	// Generate markdown content for all commands
+	for _, command := range cmd.New().Commands() {
+		// We are not generating docs for the completion command right now, as it doesn't render in Markdown correctly
+		if command.Use == "completion [bash|zsh|fish|powershell]" {
+			continue
+		}
+
+		buf := new(bytes.Buffer)
+		err := doc.GenMarkdown(command, buf)
+		if err != nil {
+			fmt.Println("Error generating markdown for command:", command.Use)
+			continue
+		}
+		mdContent += buf.String()
+	}
+
+	// Write the combined markdown content to a file
+	err := os.WriteFile(fmt.Sprintf("%s/commands.md", directory), []byte(mdContent), 0644)
```

Comment:
Does having separate files make rendering on the new docs website easier?

Reply:
So again docs website related. I actually felt it was neater to present all the commands on a single page. This is something we can discuss / consider further though 😄 .

Reply:
Sounds good. Ship it for now, and we can revisit based on feedback after the new site is up.